### PR TITLE
Add an entry for SA8775P

### PR DESCRIPTION
**Pull request by brgl**
Update the heatmap with status for sa8775p and the associated pm8654au pmic.

### CODE REVIEW

File: _pmic/pm8654au.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+pmic-gpio: 6.4
+pmic-haptics: N/A
+pmic-keypad: N/A
+pmic-labib: N/A
+pmic-lpg: N/A
+pmic-mpp: N/A
+pmic-pon: 6.4
+pmic-qnovo: N/A
+pmic-regulators: N/A
+pmic-resin: N/A
```
Comment (lumag) [inline _pmic/pm8654au.md]: At least this chip has regulators support. Granted that it has pon, I'd assume that it also has resin support. And I'd say that it is highly likely that for the platform should be a PMIC which handles LPG / PWM. Granted that you list this as the only PMIC chip, I'd ask to review this part according to the datasheet / hardware features list.

File: _soc/sa8775p.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+status-i2c: 6.4
+status-interconnects: 6.3
+status-pcie: 6.6
+status-pinctrl: 6.3
+status-powerthermal-currentlimit: N/A
+status-powerthermal-lmh: N/A
+status-powerthermal-mpm: N/A
+status-powerthermal-pdc: N/A
+status-powerthermal-spm: N/A
+status-powerthermal-tsens: N/A
```
Comment (lumag) [inline _soc/sa8775p.md]: tsens is supported upstream.

File: _soc/sa8775p.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+name: SA8775P
```
Comment (lumag) [inline _soc/sa8775p.md:2]: missing information for audio features.

File: _soc/sa8775p.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+status-spmi: 6.4
+status-sram-rpmh-stats: 6.8-next
+status-storage-nand: N/A
+status-storage-sata: N/A
+status-storage-sdcc: N/A
+status-storage-ufs: 6.5
+status-storage-ufsice: 6.7
+status-uart: 6.4
+status-usb: 6.5
+status-video-venus: N/A
```
Comment (lumag) [inline _soc/sa8775p.md]: No onboard venus decoder?

File: _soc/sa8775p.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+---
+name: SA8775P
+layout: soc
+status-clock-gcc: 6.3
+status-clock-rpmhcc: 6.3
+status-connectivity-ethernet: 6.6
```
Comment (lumag) [inline _soc/sa8775p.md:15]: No wlan/bt?

**Review by lumag — CHANGES_REQUESTED** [review]
Please review both PMIC and SoC files. Please add 'N' for hardware features which are not supported upstream.

File: _soc/sa8775p.md (hunk 3 above)
Comment (Srinivas-Kandagatla) [reply _soc/sa8775p.md:2]: No audio support yet on this.

File: _soc/sa8775p.md (hunk 4 above)
Comment (Srinivas-Kandagatla) [reply _soc/sa8775p.md]: support is not available in upstream for this yet.

File: _soc/sa8775p.md (hunk 5 above)
Comment (brgl) [reply _soc/sa8775p.md:15]: Not upstream for now, I changed it to N

File: _soc/sa8775p.md (hunk 4 above)
Comment (brgl) [reply _soc/sa8775p.md]: changed it to N

File: _pmic/pm8654au.md (hunk 1 above)
Comment (brgl) [reply _pmic/pm8654au.md]: Having trouble with my laptop ATM, but I changed the above.

File: _soc/sa8775p.md (hunk 3 above)
Comment (brgl) [reply _soc/sa8775p.md:2]: Explicitly set them to N

File: _soc/sa8775p.md (hunk 2 above)
Comment (brgl) [reply _soc/sa8775p.md]: thanks, got it

File: _pmic/pm8654au.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+---
+name: PM8654AU
```
Comment (lumag) [inline _pmic/pm8654au.md]: This should be pmm, at least according to the kernel dtsi.

Comment (brgl) [reply _pmic/pm8654au.md]: Just fixed it, thanks

**Review by lumag — APPROVED** [review]

**Pull request merged**